CONTRIBUTING.md

```diff
@@ -5,6 +5,9 @@
 - [Run Test Suite](#run-test-suite)
 
 
+## Environment Requirements
+- Node.js 15.*.*
```

Comment:
This is fairly unrealistic, node v15 will never be LTS. Could wait for node v16, which is slated for ~4/20.

Reply:
Should only be a development requirement, not a deployment requirement

Reply:
ah, my bad, missed it was in that file, 👍